api/geth_backend.go

```diff
@@ -886,18 +898,19 @@ func (b *GethStatusBackend) AppStateChange(state string) {
 				return
 			}
 		}
-	} else if s == appStateBackground && !b.forceStopWallet {
```

Comment:
How we approach this? We need the wallet running in the background so we can watch transactions